docs/architecture/grpc-for-wcf-developers/self-hosted.md

```diff
@@ -72,15 +77,15 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
 > [!NOTE]
 > If the application isn't running as a Linux service, the `UseSystemd` method doesn't do anything.
 
-Now publish your application (either framework-dependent, or self-contained for the relevant Linux runtime, e.g. `linux-x64`), either from Visual Studio by right-clicking the project and choosing *Publish* from the context menu, or from the .NET Core CLI using the following command.
+Now publish your application. The application can be either framework dependent or self-contained for the relevant Linux runtime (for example, `linux-x64`). You can publish from Visual Studio by right-clicking the project and selecting *Publish* from the shortcut menu. Or you can publish from the .NET Core CLI by using the following command.
```

Comment:
I like the change you made earlier that transformed this on a bulleted list. We should do the same here.

Reply:
Done!